### DISCUSSION TITLE
(dropIn) Store card

### DISCUSSION THREAD

**Discussion by eloy98g** [Adyen/adyen-react-native#127] [Q&A] [answered]
Hi, is there any property needed in the `configuration` object for the AdyenCheckout to store card when used?

**Comment by camil-adyen** [comment, marked as answer]
Hi @eloy98g, more information on how to store card details can be found [here](https://docs.adyen.com/payment-methods/cards/web-drop-in#store-card-details). In order to show the stored card details, you can add a configuration to the configuration of your `AdyenCheckout` instance as described [here](https://docs.adyen.com/payment-methods/cards/web-drop-in#show-stored-cards). If you want to ask consent for storing payment details, you can present a checkbox for saving the card details by setting the `enableStoreDetails` to `true` in the [configuration of the Drop-in/Components](https://docs.adyen.com/payment-methods/cards/web-drop-in#optional-configuration). 

For any other questions on Adyen specific configurations, we advice you to reach out support@adyen.com.
